doc/source/index.rst

```diff
@@ -1,13 +1,13 @@
 PyAEDT documentation  |version|
 ===============================
 
-|pyansys| |pypi| |PyPIact| |PythonVersion| |GH-CI| |codecov| |MIT| |black| |Anaconda| |pre-commit|
+|pyansys| |PyPI| |PyPIact| |PythonVersion| |GH-CI| |coverage| |MIT| |black| |Anaconda| |pre-commit|
```

Comment:
Note that the documentation for other PyAnsys libraries do not include this banner information. It's included only in the README. The then generally include what's in the "README" in the index by starting the include at a specified line.